compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/fbt/fbt-whitespace-within-text.expect.md

```diff
@@ -31,15 +31,16 @@ import fbt from "fbt";
 const _ = fbt;
 function Component(t0) {
   const $ = _c(2);
-  const { value } = t0;
   let t1;
-  if ($[0] !== value) {
+  if ($[0] !== t0) {
+    const { value } = t0;
```

Comment:
probably an artifact of MemoizeFbtOperandsInSameScope, need to double-check this